tests/test_authenticators.py

```diff
@@ -51,6 +57,8 @@ class TestAuthenticators(unittest.TestCase):
     def setUpClass(cls):
         cls.user = user = pwd.getpwuid(os.getuid())
         cls.group = group = grp.getgrgid(user.pw_gid)
+        if cls.user.pw_uid == 0 or cls.group.gr_gid == 0:
+            raise ValueError("Don't run tests as root!")
```

Comment:
Tests will fail when you run the test as root. I could work around it. Which users are guaranteed to exist on every system, root and nobody?